byte-buddy-dep/src/test/java/net/bytebuddy/implementation/auxiliary/AbstractMethodCallProxyTest.java

```diff
@@ -55,13 +60,34 @@ protected Class<?> proxyOnlyDeclaredMethodOf(Class<?> proxyTarget) throws Except
         assertThat(auxiliaryType.getDeclaredConstructors().length, is(1));
         assertThat(auxiliaryType.getDeclaredMethods().length, is(2));
         assertThat(auxiliaryType.getDeclaredFields().length, is(proxyMethod.getParameters().size() + (proxyMethod.isStatic() ? 0 : 1)));
-        int fieldIndex = 0;
-        if (!proxyMethod.isStatic()) {
-            assertThat(auxiliaryType.getDeclaredFields()[fieldIndex++].getType(), CoreMatchers.<Class<?>>is(proxyTarget));
+        if(!proxyMethod.isStatic()){
+            for (Field field : auxiliaryType.getDeclaredFields()){
+                if (field.getType() == proxyTarget){
+                	assertThat(field.getType(), CoreMatchers.<Class<?>>is(proxyTarget));
+                    break;
+                }  
+            }
         }
-        for (Class<?> parameterType : proxyTarget.getDeclaredMethods()[0].getParameterTypes()) {
-            assertThat(auxiliaryType.getDeclaredFields()[fieldIndex++].getType(), CoreMatchers.<Class<?>>is(parameterType));
+        ArrayList<Class<?>> filteredFields = new ArrayList<Class<?>>();
+        for(Field field : auxiliaryType.getDeclaredFields()) {
+        	  if (field.getType() != proxyTarget){
```

Comment:
This line seems to be indented by 8 spaces instead of 4 (what the removed lines were indented by).